### PR TITLE
perf: add min-height to improve CLS

### LINKED ISSUE #6392: [Feature]: add min-height to pages to decrease CLS
### How the project can be improved?

Our current CVV for CLS is quite bad due to the fact that we do not have height set correctly. List of products or product image appears out of nowewhere and this causes layout shift. By adding a simple CSS property to each of this pages we can decrease layout shift drastically.

### What are the acceptance criteria?

- [ ]

### Additional information

_No response_

### What version of Vue Storefront this feature can be implemented?

2.4.2

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Baroshem**
## Description

Closes #6392 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

#### Changelog
- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.

#### Tests
- [ ] I have written test cases for my code
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

> I tested manually my code, and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme

#### Code standards
- [ ] My code follows the code style of this project.

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

#### Docs
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Baroshem** [comment]
With this change we are getting about + 5-10 points for performance as CLS is decreased

File: packages/core/docs/changelog/6392.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  description: 'add min-height to improve CLS',
+  link: 'https://github.com/vuestorefront/vue-storefront/pull/6392',
```
Comment (dawid-ziobro) [inline packages/core/docs/changelog/6392.js:3]:
```suggestion
  link: 'https://github.com/vuestorefront/vue-storefront/issues/6392',
```

**Review by dawid-ziobro — APPROVED** [review]

**Comment by PiotrGrzywa** [comment]
Those changes will be manually ported to new commercetools repo.

**Comment by PiotrGrzywa** [comment]
Those changes are included in this PR:
https://github.com/vuestorefront/commercetools/pull/108

**Pull request closed without merging**